### PR TITLE
Add substack Detection

### PR DESCRIPTION
**Pull request by ysm0622**
website:
- http://substack.com/

examples:
- https://cloudgeek.substack.com/
- https://www.notboring.co/
- https://on.substack.com/
- https://news.impossible.finance/
- https://newsletter.pragmaticengineer.com/

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -4898,6 +4898,17 @@
     },
     "website": "http://subrion.com"
   },
+  "substack": {
```
Comment (nurbek91) [inline src/technologies/s.json:4901]: Substack instead substack?

File: src/technologies/s.json [hunk 2]
```diff
@@ -4898,6 +4898,17 @@
...
+    "cats": [
+      11
+    ],
+    "headers": {
```
Comment (nurbek91) [inline src/technologies/s.json:4905]: desc: "Substack is an American online platform that provides publishing, payment, analytics, and design infrastructure to support subscription newsletters."

File: src/technologies/s.json [hunk 3]
```diff
@@ -4898,6 +4898,17 @@
...
+      "x-cluster": "substack",
+      "x-served-by": "Substack"
+    },
+    "icon": "substack.png",
```
Comment (nurbek91) [inline src/technologies/s.json:4909]: [Substack.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9105653/Substack.svg.zip)
svg icon

Comment (nurbek91) [reply src/technologies/s.json:4909]: As of 2020, the minimum fee for a subscription was $5/month or $30/year, and Substack usually takes a 10 percent fee from subscription payments.

 "pricing": [
      "payg"
    ],

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate. thanks for PR. just need some additional info and replace icon

File: src/technologies/s.json (hunk 1 above)
Comment (ysm0622) [reply src/technologies/s.json]: [3d0a102](https://github.com/wappalyzer/wappalyzer/pull/6491/commits/3d0a10278d0d940780c6ffb93ce64e1ce9c9432f)

File: src/technologies/s.json (hunk 2 above)
Comment (ysm0622) [reply src/technologies/s.json:4906]: [6e2b1e2](https://github.com/wappalyzer/wappalyzer/pull/6491/commits/6e2b1e29c31a6fa2ff00e75791d63a757f3b70e9)

File: src/technologies/s.json (hunk 3 above)
Comment (ysm0622) [reply src/technologies/s.json]: [a40e44f](https://github.com/wappalyzer/wappalyzer/pull/6491/commits/a40e44f58e54b301309cd67e9073e8bd2cfd9574)

[3486f31](https://github.com/wappalyzer/wappalyzer/pull/6491/commits/3486f311140a3c2c3ce0d2cefb4b63015ae2b9e1)

**Comment by ysm0622** [comment]
Thank you for review! 🙏🏻

**Review by nurbek91 — APPROVED** [review]
Thanks mate!!

**Pull request merged**